app/components/case-studies-components/CaseStudyHeader.tsx

```diff
@@ -90,10 +90,15 @@ const CaseStudyHeader = ({ caseStudy }: MainCaseStudyProps) => {
         className="mt-6 flex lg:mt-[3.5rem]"
       >
         {caseStudy?.desktopImage && (
-          <LaptopMockup imageSrc={caseStudy?.desktopImage.image} />
+          <div className="flex w-full max-w-[80%]">
+            <LaptopMockup imageSrc={caseStudy?.desktopImage.image} />
+          </div>
         )}
```

Comment:
Why not include the div here inside the LaptopMockup component? This will help clean up the code, as you have a large code base in this one file.

Reply:
@cdsalty I did this here because in this instance the laptop and mobile mockups appear together and I want them to sit in the same div and take up 80% and 20% of the room respectively. Do you think it would be better if instead of wrapping it in a div like this I use a prop, something like 'isCaseStudyHeader' in which case I pass these styles if that is true?